src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -2949,22 +2949,24 @@ obtainAdsFromCollector (
 
     cp_resources = false;
 
+	static const char * submitter_ad_constraint = "MyType == \"" SUBMITTER_ADTYPE "\"";
+	static const char * slot_ad_constraint = "MyType == \"" STARTD_SLOT_ADTYPE "\" || MyType == \"" STARTD_OLD_ADTYPE "\"";
+
```

Comment:
*sigh*  The lines and before and after these two are space-indented, but not the rest of the function.